devito/types/dense.py

```diff
@@ -706,6 +710,9 @@ def _C_make_dataobj(self, data):
         dataobj._obj.hofs = (c_int*(self.ndim*2))(*flatten(self._offset_halo))
         dataobj._obj.oofs = (c_int*(self.ndim*2))(*flatten(self._offset_owned))
 
+        # Fields used only within C-land
+        dataobj._obj.dmap = c_void_p(0)
+
         # stash a reference to the array on _obj, so we don't let it get freed
```

Comment:
Capital letter

Reply:
oh that's a comment from the ages... will patch in the upcoming PR